doc/source/ray-overview/index.rst

```diff
@@ -0,0 +1,160 @@
+user need to do is call the class using the ``remote`` method call.
+
+.. code-block:: python
+
+    import ray
+    ray.init() # only call this one
+ 
+    @ray.remote
+    class Counter(object):
+        def __init__(self):
+            self.n = 0
+
+        def increment(self):
+            self.n += 1
+
+        def read(self):
+            return self.n
+
+    counters = [Counter.remote() for i in range(4)]
+    [c.increment.remote() for c in counters]
+    futures = [c.read.remote() for c in counters]
+    print(ray.get(futures)) # [1, 1, 1, 1]
+
+
+An Overview of the Ray Libraries
+================================
+
+Ray has a vast ecosystem of libraries and frameworks built on top of it. The main ones being:
+
+- :ref:`tune-index`
+- :ref:`rllib-index`
+- :ref:`sgd-index`
+- :ref:`rayserve`
+
+
+Tune Quick Start
+----------------
+
+`Tune`_ is a library for hyperparameter tuning at any scale. With Tune, you can launch a multi-node distributed hyperparameter sweep in less than 10 lines of code. Tune supports any deep learning framework, including PyTorch, TensorFlow, and Keras.
+
+.. note::
+
+    To run this example, you will need to install the following:
+
+    .. code-block:: bash
+
+        $ pip install ray torch torchvision filelock
+
+
+This example runs a small grid search to train a CNN using PyTorch and Tune.
+
+.. literalinclude:: ../../python/ray/tune/tests/example.py
+   :language: python
+   :start-after: __quick_start_begin__
+   :end-before: __quick_start_end__
+
+If TensorBoard is installed, automatically visualize all trial results:
+
+.. code-block:: bash
+
+    tensorboard --logdir ~/ray_results
+
+.. _`Tune`: tune.html
+
+RLlib Quick Start
+-----------------
+
+`RLlib`_ is an open-source library for reinforcement learning built on top of Ray that offers both high scalability and a unified API for a variety of applications.
+
+.. code-block:: bash
+
+  pip install tensorflow  # or tensorflow-gpu
+  pip install ray[rllib]  # also recommended: ray[debug]
+
+.. code-block:: python
+
+    import gym
+    from gym.spaces import Discrete, Box
+    from ray import tune
+
+    class SimpleCorridor(gym.Env):
+        def __init__(self, config):
+            self.end_pos = config["corridor_length"]
+            self.cur_pos = 0
+            self.action_space = Discrete(2)
+            self.observation_space = Box(0.0, self.end_pos, shape=(1, ))
+
+        def reset(self):
+            self.cur_pos = 0
+            return [self.cur_pos]
+
+        def step(self, action):
+            if action == 0 and self.cur_pos > 0:
+                self.cur_pos -= 1
+            elif action == 1:
+                self.cur_pos += 1
+            done = self.cur_pos >= self.end_pos
+            return [self.cur_pos], 1 if done else 0, done, {}
+
+    tune.run(
+        "PPO",
+        config={
+            "env": SimpleCorridor,
+            "num_workers": 4,
+            "env_config": {"corridor_length": 5}})
+
+.. _`RLlib`: rllib.html
+
```

Comment:
Add serve quick start?

Reply:
I didn't add these, would like to address in a follow PR.